modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryLocalServiceTest.java

```diff
@@ -1039,6 +1039,12 @@ private void _testGetUserAccountEntries(
 			userAccountEntries.toString(), expectedAccountEntries.size(),
 			userAccountEntries.size());
 
+		Assert.assertEquals(
+			userAccountEntries.toString(), expectedAccountEntries.size(),
```

Comment:
do we need the message param? It seems odd to use the `userAccountEntries` as message as it's not used in the assertion.